test/integration/team.js

```diff
@@ -1,22 +1,116 @@
 import chai from 'chai';
-
-import { Dropbox } from '../../index.js';
+import fs from 'fs/promises';
+import path from 'path';
+import { Dropbox, DropboxAuth } from '../../index.js';
 import { DropboxResponse } from '../../src/response.js';
 
-describe('Team', () => {
-  const dbx = new Dropbox({ accessToken: process.env.DROPBOX_TEAM_TOKEN });
+const appInfo = {
+  LEGACY: {
+    token: process.env.LEGACY_TEAM_DROPBOX_TOKEN,
+    app_key: process.env.LEGACY_TEAM_APP_KEY,
+    app_secret: process.env.LEGACY_TEAM_APP_SECRET,
+    refresh_token: process.env.LEGACY_TEAM_REFRESH_TOKEN,
+  },
+  SCOPED: {
+    token: process.env.SCOPED_TEAM_DROPBOX_TOKEN,
+    app_key: process.env.SCOPED_TEAM_APP_KEY,
+    app_secret: process.env.SCOPED_TEAM_APP_SECRET,
+    refresh_token: process.env.SCOPED_TEAM_REFRESH_TOKEN,
+  },
+};
+
+for (const appType in appInfo) {
+  if (appType) { // for linter
+    describe(`Team ${appType}`, () => {
+      const dbxAuth = new DropboxAuth({
```

Comment:
Could we just say 
```
const dbxAuth = new DropboxAuth(appInfo[appType])
```
I feel we should keep naming consistent as well. The sdk calls them `clientId` so we should not be refering to it as `app_key`.